ted_sws/core/entrypoints/api/main.py

```diff
@@ -0,0 +1,124 @@
+import hashlib
+import logging
+import subprocess
+import uuid
+from enum import Enum
+from typing import Optional, Union, Tuple
+
+import click
+from fastapi import FastAPI
+
+from ted_sws.event_manager.adapters.logger import Logger, LOG_INFO_LEVEL
+from ted_sws.event_manager.domain.message_bus import message_bus
+from ted_sws.event_manager.model.message import Log
+
+
+class ResponseType(Enum):
+    JSON = "json"
+    RAW = "raw"
+
+
+class UUIDInputProcessType(Enum):
+    MD5 = "md5"
+    RAW = "raw"
+
+
+class UUIDNamespaceType(Enum):
+    DNS = "DNS"
+    URL = "URL"
+    OID = "OID"
+    X500 = "X500"
+
+
+API_SERVER_LOG_LEVELS: Tuple = ('critical', 'error', 'warning', 'info', 'debug', 'trace')
+
+app = FastAPI()
+
+
+@click.command()
+@click.option('-h', '--host', default="localhost")
+@click.option('-p', '--port', default=8000, type=int)
+@click.option('-l', '--log-level', default="info", type=click.Choice(API_SERVER_LOG_LEVELS))
+def api_server_start(host, port, log_level):
+    logger = Logger(name="API_SERVER", level=LOG_INFO_LEVEL)
+    fmt = "[%(asctime)s] - %(name)s - %(levelname)s:\n%(message)s"
+    date_fmt = "%Y-%m-%d %H:%M:%S"
+    logger_formatter = logging.Formatter(fmt, date_fmt)
+    logger.add_stdout_handler(formatter=logger_formatter)
+    log_level_text = f" --log-level {log_level}" if log_level else ""
+    bash_script = f"uvicorn --host {host} --port {port}{log_level_text} ted_sws.core.entrypoints.api.main:app --reload"
```

Comment:
oh! surely there is a better way to do this.

Reply:
we do not need to start a bash command, we can leave this aspect to the infra part. See how `gunicorn` is used in MDR projects.

Reply:
It is meant for local usage.